sdk/keyvault/azure_security_keyvault_secrets/tests/secret_client.rs

```diff
@@ -200,3 +207,68 @@ async fn purge_secret(ctx: TestContext) -> Result<()> {
 
     Ok(())
 }
+
+#[recorded::test]
+async fn round_trip_secret_verify_telemetry(ctx: TestContext) -> Result<()> {
+    use azure_core_test::tracing::test_instrumentation_for_api;
```

Comment:
`test_instrumentation_for_api`? Looks like something that was meant to be internal leaked out - mainly, the "for_api". Can we just call it `test_instrumentation`? Concise and descriptive.